Elaborate replacement of PLA gantry parts

tehmaze opened the pull request:
Elaborated on replacing the PLA parts as per issue #91

tehmaze committed:
Update disassemble.md

Elaborated on replacing the PLA parts as per issue #91

tehmaze committed:
Merge branch 'gh-pages' into issue-91

HumbleDeer commented on docs/vz330_mellow/gantry/disassemble.md:
```diff
@@ -9,7 +9,13 @@ has_children: false
+The Mellow VzBoT kit comes with the gantry pre-assembled containing printed parts made out of PLA 
+plastic. While the PLA plastic parts are a good starting point and provide you with a clear
+reference as to how the gantry is assembled, it is highly recommended to replace the supplied PLA
+parts with a different material that can withstand higher temperatures. Te PLA parts may warp 
```
typo here :)
te -> the

HumbleDeer replied:
Oh -- Having the artificial line length limits isn't really required for our use cases. Any particular reason for using them? 't makes the diff look funny :^)

HumbleDeer reviewed (changes requested):
All that remains would be to copy these changes to the other manuals that have a disassembly/replacement step in them. 

Willing to do this myself too, no big deal.

tehmaze committed:
Update disassemble.md

Typofix

tehmaze commented:
> All that remains would be to copy these changes to the other manuals that have a disassembly/replacement step in them.

Can we make that a snippet that can be recycled in other places?

HumbleDeer commented:
> > All that remains would be to copy these changes to the other manuals that have a disassembly/replacement step in them.
> 
> Can we make that a snippet that can be recycled in other places?

yesn't.

Using our system it's not worth doing that. We're better off keeping track of where we have these things posted by having the same stuff posted on the same pages with the same titles.

tehmaze committed:
Added note about PLA parts in other places as per @HumbleDeers suggestion

HumbleDeer reviewed (approved):
It's not ideal, but we can revise this later on or in the future. It's a start! I'll let you decide whether you want to merge this in or wait and polish it up a little more.

Commit:
Create placeholder parts page on vz235_mellow

Commit:
Change warning to placeholder-parts reference

HumbleDeer commented:
Please see https://github.com/tehmaze-labs/vzbot-docs/pull/2 before merging this PR

Commit:
Create placeholder parts page on vz330_mellow

This one was originally called "disassemble", but was renamed to unify the titles between the manuals.

tehmaze committed:
Merge pull request #2 from HumbleDeer/issue-91

Create placeholder parts page on vz235_mellow
